modules/renter/renter.go

```diff
@@ -165,8 +170,7 @@ type Renter struct {
 	//
 	// tracking contains a list of files that the user intends to maintain. By
 	// default, files loaded through sharing are not maintained by the user.
-	files    map[string]*file
-	tracking map[string]trackedFile // Map from nickname to metadata.
+	files map[string]*file
```

Comment:
Haha that is actually something I was wondering about when working on refactoring the file format. I'm glad that it's gone now.

Reply:
It's not gone it just got moved